<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PO3T4V3" failures="9" tests="65" name="testCases" time="893.072" errors="0" timestamp="2022-07-21T10:59:42 AMT">
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLabelsPresenceVerification_Test" time="0.543"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpLabelsPresenceVerification_Test" time="0.201"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterLabelsPresenceVerification_Test" time="0.516"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogoPresenceVerification_Test" time="0.055"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpLogoPresenceVerification_Test" time="0.024"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterLogoPresenceVerification_Test" time="0.168"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpCloseButtonFunctionality_Test" time="0.687"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpSupportLink_Test" time="0.35">
    <failure type="java.lang.AssertionError" message="did not expect [https://craftbet.com/forgot-password] but found [https://craftbet.com/forgot-password]">
      <![CDATA[java.lang.AssertionError: did not expect [https://craftbet.com/forgot-password] but found [https://craftbet.com/forgot-password]
at testCases.PasswordRecoveryTest.RecoveryPopUpSupportLink_Test(PasswordRecoveryTest.java:84)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- RecoveryPopUpSupportLink_Test -->
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterRegisterByDropDownVerification_Test" time="0.475"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpEyeButtonFunctionality_Test" time="0.369"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpPositiveTest" time="0.557"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterCurrencyDropDownVerification_Test" time="0.646"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpRememberMeLabelClickAbility_Test" time="0.213"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="20.59">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.PasswordRecoveryTest.RecoveryPopUpNegativeTest(PasswordRecoveryTest.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- RecoveryPopUpNegativeTest -->
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="20.607">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.PasswordRecoveryTest.RecoveryPopUpNegativeTest(PasswordRecoveryTest.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- RecoveryPopUpNegativeTest -->
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.299"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.353"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.371"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.372"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.313"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.373"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.389"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.366"/>
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="20.599">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.PasswordRecoveryTest.RecoveryPopUpNegativeTest(PasswordRecoveryTest.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- RecoveryPopUpNegativeTest -->
  <testcase classname="testCases.PasswordRecoveryTest" name="RecoveryPopUpNegativeTest" time="40.392"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpCloseButtonFunctionality" time="0.635"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterRegisterPrivacyPolicy" time="0.312"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpCheckBoxClickAbility_Test" time="0.223"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpTermsAndConditionsLinkFunctionality" time="0.446"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpForgotPasswordLink_Test" time="0.144"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpLogInLinkFunctionality" time="0.152"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpSignUpLink_Test" time="0.228"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickFullRegistrationButtonFunctionality" time="0.316"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInPositiveTest" time="10.358">
    <failure type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;xpath&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;//*[@class=&amp;apos;user_id&amp;apos;]&amp;quot;}
  (Session info: headless chrome=103.0.5060.114)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: &amp;apos;4.2.1&amp;apos;, revision: &amp;apos;ac4d0fdd4a&amp;apos;
System info: host: &amp;apos;DESKTOP-PO3T4V3&amp;apos;, ip: &amp;apos;10.50.17.113&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.2&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [62d6a0687b5e1afc9761a345f94595fc, findElement {using=xpath, value=//*[@class=&amp;apos;user_id&amp;apos;]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 103.0.5060.114, chrome: {chromedriverVersion: 103.0.5060.134 (8ec6fce403b..., userDataDir: C:\Users\NERSES~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:59801}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:59801/devtoo..., se:cdpVersion: 103.0.5060.114, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 62d6a0687b5e1afc9761a345f94595fc">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@class='user_id']"}
  (Session info: headless chrome=103.0.5060.114)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.2.1', revision: 'ac4d0fdd4a'
System info: host: 'DESKTOP-PO3T4V3', ip: '10.50.17.113', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [62d6a0687b5e1afc9761a345f94595fc, findElement {using=xpath, value=//*[@class='user_id']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 103.0.5060.114, chrome: {chromedriverVersion: 103.0.5060.134 (8ec6fce403b..., userDataDir: C:\Users\NERSES~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:59801}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:59801/devtoo..., se:cdpVersion: 103.0.5060.114, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 62d6a0687b5e1afc9761a345f94595fc
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:551)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:369)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:361)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
at jdk.proxy2/jdk.proxy2.$Proxy19.isEnabled(Unknown Source)
at pageObjects.BasePage.elementIsEnable(BasePage.java:102)
at pageObjects.CraftBet_Header_Page.userIdLabelIsEnabled(CraftBet_Header_Page.java:205)
at testCases.LoginPopUpTest.loginPopUpLogInPositiveTest(LoginPopUpTest.java:187)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- loginPopUpLogInPositiveTest -->
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.32"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.344"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.371"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.379"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.381"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.379"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.309"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.305"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpLogInNegativeTest" time="10.328"/>
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpErrorMessageValidationInvalidUsername" time="1.369">
    <failure type="java.lang.AssertionError" message="expected [Username or Password is incorrect. Please try again.] but found [Younger]">
      <![CDATA[java.lang.AssertionError: expected [Username or Password is incorrect. Please try again.] but found [Younger]
at testCases.LoginPopUpTest.loginPopUpErrorMessageValidationInvalidUsername(LoginPopUpTest.java:259)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- loginPopUpErrorMessageValidationInvalidUsername -->
  <testcase classname="testCases.LoginPopUpTest" name="loginPopUpErrorMessageValidationInvalidPassword" time="1.371"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailPositiveTest" time="2.584"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="0.993"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="0.871"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="1.014"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="0.985"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="1.002">
    <failure type="java.lang.AssertionError" message="expected [Invalid email address] but found [Email address must be unique]">
      <![CDATA[java.lang.AssertionError: expected [Invalid email address] but found [Email address must be unique]
at testCases.SignUpQuickRegisterTest.SignUpPopUpQuickRegisterWithEmailNegativeTest(SignUpQuickRegisterTest.java:256)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- SignUpPopUpQuickRegisterWithEmailNegativeTest -->
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="0.936"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="0.889">
    <failure type="java.lang.AssertionError" message="expected [Invalid email address] but found [Email address must be unique]">
      <![CDATA[java.lang.AssertionError: expected [Invalid email address] but found [Email address must be unique]
at testCases.SignUpQuickRegisterTest.SignUpPopUpQuickRegisterWithEmailNegativeTest(SignUpQuickRegisterTest.java:256)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- SignUpPopUpQuickRegisterWithEmailNegativeTest -->
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="1.033"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithEmailNegativeTest" time="0.854"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobilePositiveTest" time="3.626"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.413"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.487"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.459"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.447"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.426"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.491"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.413"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.504"/>
  <testcase classname="testCases.SignUpQuickRegisterTest" name="SignUpPopUpQuickRegisterWithMobileNegativeTest" time="1.425">
    <failure type="java.lang.AssertionError" message="expected [craft_btn reg_btn not_active] but found [craft_btn reg_btn active-item]">
      <![CDATA[java.lang.AssertionError: expected [craft_btn reg_btn not_active] but found [craft_btn reg_btn active-item]
at testCases.SignUpQuickRegisterTest.SignUpPopUpQuickRegisterWithMobileNegativeTest(SignUpQuickRegisterTest.java:322)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- SignUpPopUpQuickRegisterWithMobileNegativeTest -->
</testsuite> <!-- testCases -->
